### DISCUSSION TITLE
Can't login with my office account because of same phone number

### DISCUSSION THREAD

**Discussion by ODelibalta** [vercel/community#4263] [Account Issue] [answered]
### Question

I have two Vercel accounts, one personal, one for work. 

Can't login to my work account because it asks me to enter a phone number to have the code sent to. I can't enter my cell because it errors saying that the number is in use by another account... 

Not sure what to do at this point.

**Comment by amyegan** [comment, marked as answer]
Sorry you're having trouble with your second account. 🙁 We can't safely discuss account information in public because of security concerns. Please email [registration@vercel.com](mailto:registration@vercel.com) to open a ticket with the team that can help with login issues. 

Be sure to let them know about the email addresses and phone number used for your accounts and how you plan to use each account (i.e. one is for work and the other is for personal projects). That will help speed up the verification process. 🙏
